lib/pharos/addon_manager.rb

```diff
@@ -95,8 +98,6 @@ def each(&block)
         addon = addon_class.new(config, enabled: true, **options)
         addon.validate
         yield_addon_with_retry(addon, &block)
-        post_install_message = addon.post_install_message
-        @cluster_context['post_install_messages'][addon.name] = post_install_message if post_install_message
```

Comment:
moved post install message handling to `ClusterManager` because `each` method is not only for install